packages/prisma/migrations/20240801210911_adds_per_user_and_team_feature_flags/migration.sql

```diff
@@ -0,0 +1,35 @@
+-- CreateTable
+CREATE TABLE "_user_features" (
```

Comment:
These names are weird and not standard with our naming conventions. Why don't we make new models in the schema so we get better and more consistent naming? 
e.g. `UserFeature` with columns `Id`, `UserId`, `FeatureId`

Reply:
These are autogenerated from the prisma schema changes. IDK if customizing these would affect the type generator

Reply:
Tinkering into this